afu-test/afu_test.h

```diff
@@ -189,6 +189,7 @@ class afu {
     }
 
     auto tokens = fpga::token::enumerate({filter});
+    perf_token_ = tokens[0];
```

Comment:
Hi Ramya, this change is causing a segfault in test_dummy_afu.  Could you please implement a fix?  Thanks!

Reply:
getting a token from handler. Its fixed testing error